package.json

```diff
@@ -72,6 +72,6 @@
     "flow-bin": "^0.50.0"
   },
   "dependencies": {
-    "uuid": "^2.0.1"
+    "uuid": "^8.3.2"
```

Comment:
Jumping 6 major releases in a dependency seems like a questionable thing to do in a bugfix update. This type of change should probably also involve a major bump.

Can you do that, and prepare the [CHANGELOG](https://github.com/bvaughn/js-worker-search/blob/master/CHANGELOG.md) notes as well?

Reply:
> We're trying to use this package with vitejs

Also, do you mean the vitejs project itself? Or an app you're building that is built with vitejs? Just curious.

Reply:
Thanks for the quick reply!
I didn't see a reason not to upgrade to the latest as the change is so minor, but I'm happy to change to another version if you have anything in mind :)

Reply:
> Also, do you mean the vitejs project itself? Or an app you're building that is built with vitejs? Just curious.

An app that we're building - https://github.com/spinnaker/deck

Reply:
@bvaughn, is this good to go? Please let us know if you want anything changed.

Reply:
Sorry. Dropped this thread.